lib/src/components/NxRadio/NxRadio.scss

```diff
@@ -10,52 +10,78 @@
 @import '../../scss-shared/nx-input-helpers';
 
 .nx-radio {
-  $circle-width: 16px;
-  $label-margin: 8px;
-  $line-height: 20px;
-
   @include container-horizontal;
   @include regular();
   @include focusable-toggle-input(".nx-radio__outer-circle");
 
-  display: inline-block;
-  font-size: 14px;
-  line-height: $line-height;
+  display: block;
```

Comment:
With this change they're _always_ going to be block.  I was expecting the width adjustment to be done in the context of a fieldset.

Reply:
> With this change they're _always_ going to be block. I was expecting the width adjustment to be done in the context of a fieldset.

Yes, I think they should always be block. There are exceedingly few instances where we would want `NxRadio` to be inline, so I think it's better to make them block by default and if there's a case where they are inline that should be custom.

Reply:
> > With this change they're _always_ going to be block. I was expecting the width adjustment to be done in the context of a fieldset.
> 
> Yes, I think they should always be block. There are exceedingly few instances where we would want `NxRadio` to be inline, so I think it's better to make them block by default and if there's a case where they are inline that should be custom.

The same will be true of `NxCheckbox`.

Reply:
> Yes, I think they should always be block. There are few instances where we would want NxRadio to be inline, so I think it's better to make them block by default and if there's a case where they are inline that should be custom.

> The same will be true of NxCheckbox.

Be aware that my original reason for making them inline was for consistency with natively styled radios and checkboxes.